### DISCUSSION TITLE
Html Input set password type, but it can not constraint input in English.

### DISCUSSION THREAD

**Discussion by WinderChou** [cefsharp/CefSharp#3735] [General]
Hi,

I have an question.
I set html input to password type, it could constraint to input in English before version.
But current version is removed this function like. 
How can I set this function in the current version please?
Thanks a lot.

Winder

**Comment by amaitland** [comment]
Have you compared the behaviour to Chrome? 

> How can I set this function in the current version please?

It's not clear what function you are referring to.
